### PR TITLE
ft: S3C-1561 - get storage untilized by a resource

### PR DESCRIPTION
**Pull request by anurag4DSB**
**What does this PR do, and why do we need it?**
Supports the new feature of Quotas in vault by getting the storage Utilized of a resource.

**Which issue does this PR fix?**
S3C-1561

**Special notes for your reviewers:**
This is helper code to get storage utilized by a resource for Quotas.

### CODE REVIEW

File: lib/UtapiClient.js [hunk 1]
```diff
@@ -971,6 +971,28 @@ class UtapiClient {
...
+     * @param {string} [params.bucket] - (optional) bucket name
+     * @param {string} [params.accountId] - (optional) account canonical ID
+     * @param {string} [params.userId] - (optional) account ID
+     * @param {string} [params.level] - (required) level of granularity
+     * @param {string} [params.service] - (required) service name (s3 for ex.)
+     * @param {object} log - Werelogs request logger
+     * @param {callback} callback - callback to call
+     * @return {undefined}
+     */
+    getStorageUtilized(params, log, callback) {
```
Comment (JianqinWang) [inline lib/UtapiClient.js]: The function passes in a `log` param but it's not used
Should there be a log statement before the `InternalError` so we don't mask the error completely?

**Comment by anurag4DSB** [comment]
@bert-e reset

**Review by JianqinWang — APPROVED** [review]

**Review by vrancurel — APPROVED** [review]

**Comment by anurag4DSB** [comment]
@bert-e approve

**Review by bennettbuchanan — COMMENTED** [review]
Let’s be sure to add unit and functional tests to help with maintainability.

**Pull request merged**